modules/mapper-extras/src/test/java/org/elasticsearch/index/mapper/extras/RankFeatureFieldMapperTests.java

```diff
@@ -125,6 +128,44 @@ public void testNegativeScoreImpact() throws Exception {
         assertTrue(freq1 > freq2);
     }
 
+    /**
+     * Method that tests an exception is thrown when nullValue provided for rank feature is not a positive number
+     */
+    public void testExceptionIsThrownWhenNullValueNonPositive() {
+        MapperParsingException e = expectThrows(
+            MapperParsingException.class,
+            () -> createDocumentMapper(fieldMapping(b -> b.field("type", "rank_feature").field("null_value", "-2")))
+        );
+        String message = "[null_value] must be a positive normal float for field of type [rank_feature], got "
+            + Float.valueOf(-2f).toString()
+            + " which is less than the minimum positive normal float: "
+            + Float.MIN_NORMAL;
+        assertEquals(RestStatus.BAD_REQUEST, e.status());
+        assertEquals(IllegalArgumentException.class, e.getCause().getClass());
+        assertEquals(message, e.getCause().getMessage());
+    }
+
+    public void testNullValue() throws IOException {
+        DocumentMapper mapper = createDocumentMapper(fieldMapping(this::minimalMapping));
```

Comment:
Sorry, Lines 149-151 are not relevant for `rank_feature` field, which we need to explicitly define in mappings.

Reply:
Should I delete those lines or replace them with these lines of code?
 `DocumentMapper mapper = createDocumentMapper(fieldMapping(b -> b.field("type", "rank_feature")));
        ParsedDocument doc = mapper.parse(source(b -> b.nullField("field")));
        assertThat(doc.rootDoc().getFields("_feature"), empty());
`